koa-crud/src/data-access/mongoose/members/actions.ts

```diff
@@ -1,31 +1,63 @@
-import { MembersStore } from '../../../types/index';
-import { Member as MemberModel } from '../../../lib/mongoose/models/member';
+import MemberModelType, {
+  Member,
+  MemberDocument,
+} from '../../../lib/mongoose/models/member';
 
-const actions = ({ Member }: { Member: typeof MemberModel }): MembersStore => {
+type Information = {
+  username?: Member['username'];
+  password?: Member['password'];
+  realName?: Member[`realName`];
+  email?: Member[`email`];
+  bankAccount?: Member[`bankAccount`];
+  balance?: Member[`balance`];
+};
```

Comment:
Can you rename this type to `MemberInformation`, just for the sake of distinction?

And you can refactor this to:
```ts
type MemberInformation = Partial<Omit<Member, 'omit not needed fields'>>
```

Reply:
noted on this